### ISSUE TITLE
Library ReadMe/Description

### ISSUE THREAD

**Issue by RourkeDawson** [joshuanario/digits#5]
Thanks for taking the time to understand and review the inner workings of `digits`.  Now that I'm reviewing this test case, this is not correct behavior for `digits`.  This test case should modified and updated.

https://github.com/joshuanario/digits/blob/d391bf4b7f99bfce9b994f8104f9082715491da2/digits_test.go#L29-L44

This test case should be updated with the following specifications:

```
 	stimulus: Stimulus{ 
 		precision:           digits.Oneth, 
 		value:               "0.99", 
 		groupSeparator:      ',', 
 		fractionalPrecision: digits.PreserveUpToHundredth, 
 	}, 
 	expectation: Expectation{ 
 		sigFigs:    "0.", 
 		nonSigFigs: "99", 
 		strOut:     "0.99", 
 		head:       "", 
 		core:       "0.",   //decimal points (or separators) should always be in core if the precisions is in the oneth place (this also holds true for EXACT precisions) and should not show up in tail for those specific conditions
 		tail:       "99", 
 	}, 

```

_Originally posted by @joshuanario in https://github.com/joshuanario/digits/issues/4#issuecomment-1806834418_
            
Some description should be added to the library's readme to disclose the intended use and expected results for output w/ regard to significant figures. intended use will almost always result in spurious digits, as despite designated precision, the actual significant figure displayed is determined by the decimal preservation.

see: https://en.wikipedia.org/wiki/Significant_figures

**Comment by joshuanario** [comment]
Yes.  A well written README.md will go along ways now that this project is getting alot of visibility.  Thanks for creating the issue.  Do you want me to make the README or are you willing to work on it? @SpencerDawson

**Comment by RourkeDawson** [comment]
> Yes. A well written README.md will go along ways now that this project is getting alot of visibility. Thanks for creating the issue. Do you want me to make the README or are you willing to work on it? @SpencerDawson

I have a PR open for this, #9 that mostly addresses contributing. I'll open a second PR for this specifically, and I can template it out for you, but since this is your library, you might have better insight to fill out the details. I'll @ you here again when it's ready.
